pkgs/development/python-modules/mask-rcnn/default.nix

```diff
@@ -0,0 +1,51 @@
+{ buildPythonPackage
+, cython
+, fetchFromGitHub
+, h5py
+, imgaug
+, ipython
+, Keras
+, lib
+, matplotlib
+, numpy
+, opencv3
+, pillow
+, scikitimage
+, scipy
+, tensorflow
+}:
+
+buildPythonPackage rec {
+  pname = "mask-rcnn";
+  version = "2.1";
+  
+  src = fetchFromGitHub {
+    owner = "matterport";
+    repo = "Mask_RCNN";
+    rev = "3deaec5d902d16e1daf56b62d5971d428dc920bc";
+    sha256 = "13s3q9yh2q9m9vyksd269mww3bni4q2w7q5l419q70ca075qp8zp";
+  };
+      
+  propagatedBuildInputs = [
+    imgaug
+    numpy
+    scipy
+    pillow
+    cython
+    matplotlib
+    scikitimage
+    Keras
+    opencv3
+    h5py
+    ipython
+    tensorflow
```

Comment:
not related to PR, but this is like a list of "most likely to be broken" packages.... xD.

Tensorflow especially lately.